### ISSUE TITLE
Provide enriched capability to interact with alert

### ISSUE THREAD

**Issue by tomkerkhove** [microsoft/AzureAppForMicrosoftTeams#10]
We use the following example at some customers:
![alert-sample](https://user-images.githubusercontent.com/4345663/78758754-e7cc7d80-797e-11ea-99c1-0e99709165cd.gif)

Are there plans to bring same capability to
- See timeline of alert with history
- Have capability to to go to resource
- Have capability to to go to alert
- Have capability to see contextual information coming from Azure Tags?

**Comment by divais** [comment]
Hi @tomkerkhove 
Have capability to to go to resource and alert are already available. 

Alert timeline isnt available in the same card but available in the channel. When threading is available for the alerts, the timeline will be more clear. 

Can you please you are using this with which customer?

**Comment by tomkerkhove** [comment]
That's correct but I'm more wondering if there are plans to make it more interactive as in the GIF above?

_PS: I work for Codit_
